ckanext/benap/benap_schema.json

```diff
@@ -422,6 +423,27 @@
       "benap_helper_ontology": "NUTS1_BE",
       "required": "true"
     },
+    {
+      "field_name": "network_coverage",
+      "label": {
+        "en": "Network Coverage",
+        "fr": "Couveerture réseau",
+        "nl": "Netwerkdekking",
+        "de": "Netzabdeckung"
+      },
+      "display_snippet": "select.html",
+      "display_property": null,
```

Comment:
display_property kan weggelaten worden aangezien het toch null is??

Reply:
Deleted